README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
It's unclear what 'task' refers to. Provide a brief description or context about it to avoid confusion for new users.